src/tools/rustbook/src/main.rs

```diff
@@ -64,10 +72,32 @@ fn main() {
                 }
             };
         },
+        ("linkcheck", Some(sub_matches)) => {
+            if let Err(broken_links) = linkcheck(sub_matches) {
```

Comment:
Could we rename `broken_links` to `err`? It is not guaranteed to be a broken links error; there are a bunch of other error types: https://docs.rs/mdbook-linkcheck/0.2.3/mdbook_linkcheck/errors/index.html.